artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java

```diff
@@ -120,6 +120,7 @@ public class ActiveMQActivationSpec extends ConnectionFactoryProperties implemen
    /**
     * Transaction timeout
     */
+   @Deprecated
```

Comment:
If it is deprecated for future removal as indicated in the documentation, should the Deprecated tag indicate that with the forRemoval flag?

Reply:
Done.